src/foam/u2/ExportModal.js

```diff
@@ -0,0 +1,145 @@
+foam.CLASS({
+  package: 'foam.u2',
+  name: 'ExportModal',
+  extends: 'foam.u2.View',
+
+  documentation: 'Export Modal',
+
+  requires: [
+    'foam.u2.ModalHeader',
+    'foam.nanos.export.JSONDriver',
```

Comment:
Isn't the list of export drivers stored in a DAO, which would make it more extensible than hard-coding it here?